# SNPUTILS---Implementing-GWAS
Understanding the genetic basis of complex traits and diseases is a fundamental challenge in modern biology. Many traits, such as height, intelligence, and susceptibility to common diseases, are influenced by multiple genetic variants with small effect sizes. Genome-wide association studies (GWAS) have emerged as a powerful approach to identify these genetic variants by scanning the entire genome for associations with specific traits or diseases. By examining a large number of single nucleotide polymorphisms (SNPs) across many individuals, GWAS can pinpoint genetic variations that contribute to phenotypic diversity.
In this study, we aim to identify genetic loci associated with a quantitative trait using simulated genotype and phenotype data. This approach allows us to rigorously test our GWAS pipeline in a controlled environment, providing confidence in its applicability to real-world datasets. We utilized Python-based libraries, including pandas, numpy, statsmodels, scipy, matplotlib, seaborn, and sklearn, to perform data preprocessing, quality control, statistical analysis, and visualization.
